core/src/test/scala/unit/kafka/admin/ConfigCommandTest.scala

```diff
@@ -736,6 +731,12 @@ class ConfigCommandTest extends ZooKeeperTestHarness with Logging {
     verifyAlterBrokerConfig(node, "", List("--entity-default"))
   }
 
+  @Test
+  def shouldAddBrokerDynamicConfig(): Unit = {
+    val node = new Node(1, "localhost", 9092)
+    verifyAlterBrokerConfig(node, "1", List("--entity-name", "1"))
+  }
```

Comment:
reorder this test case to put the `verifyAlterBrokerConfig` test suits together. Note: the previous test is also test with `verifyAlterBrokerConfig`